infra/src/main/kotlin/fi/fta/geoviite/infra/geocoding/GeocodingController.kt

```diff
@@ -25,18 +27,18 @@ class GeocodingController(
 ) {
     private val logger: Logger = LoggerFactory.getLogger(this::class.java)
 
-    @PreAuthorize(AUTH_UI_READ)
-    @GetMapping("/{publishType}/address/{trackNumberId}")
+    @PreAuthorize(AUTH_VIEW_DRAFT_OR_OFFICIAL_BY_PUBLISH_TYPE)
+    @GetMapping("/{$PUBLISH_TYPE}/address/{trackNumberId}")
     fun getTrackAddress(
-        @PathVariable("publishType") publishType: PublishType,
+        @PathVariable("$PUBLISH_TYPE") publishType: PublishType,
```

Comment:
Tässä näkee vähän tuon oikkavalintahäkkyrän käyttöä. Aina jos näkee jossain tuon `AUTH_VIEW_DRAFT_OR_OFFICIAL_BY_PUBLISH_TYPE`-authin, niin sen yhteydessä pitäisi näkyä vähintään kaksi `$PUBLISH_TYPE`-vakion käyttöä (urlissa ja pathvariablessa.) Jos näin ei ole niin menee väärin ja siitä tulee älähtää (reviewillä, koodihan ei sitä huomaa)

Reply:
Tällaisessa myös minihuomiona, että lainausmerkkejä ei varmaankaan tarvita ellei @PathVariable sisällä myös jotain muuta?